packages/commonjs/test/cjs-module-system.spec.ts

```diff
@@ -317,4 +322,51 @@ module.exports = global;`,
 
     expect(requireModule(sampleJsonPath)).to.eql({ name: 'test' });
   });
+
+  it('allows registering to evaluation hooks', () => {
+    const aFile = '/a.js';
+    const bFile = '/b.js';
+    const fs = createMemoryFs({
+      [aFile]: `require('./b');`,
+      [bFile]: 'module.exports = 5;',
+    });
+    const callArray: string[] = [];
+    const { requireModule } = createCjsModuleSystem({
+      fs,
+      loadModuleHook: (requireModule) => (modulePath) => {
```

Comment:
rename `requireModule` to `loadModule`. matches signature and doesn't overshadow `requireModule` from two lines up.